addons/website_sale/static/src/js/website_sale.js

```diff
@@ -163,6 +163,33 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
 
         this.isWebsite = true;
     },
+    /**
+     * @override
+     */
+    willStart: function () {
+        var self = this;
+        return this._super().then(function (el) {
+            var hash = window.location.hash.substring(1);
+            if (hash) {
+                hash = decodeURIComponent(hash).split(',');
+                var inputs = self.$el.find('input.js_variant_change, select.js_variant_change option');
+                _.each(hash, function (elem) {
+                    var toSelect = inputs.filter('[data-value_id="' + elem + '"]');
+                    if (toSelect.is('input[type="radio"]')) {
+                        if (toSelect.is('[data-attribute_name="Color"]')) {
+                            toSelect.trigger('click');
+                        } else {
+                            toSelect.attr('checked', true);
```

Comment:
`attr('checked', true)` is not valid, it works by chance.

It is either
- `attr('checked', 'checked')` to write the DOM
or
- `prop('checked', true)` to only toggle the input state

The second solution is the correct one... except that the new editor forces us to do the first one to continue working (otherwise all the states are lost when entering edit mode). I would do both explicitely to avoid forgetting about it. So:
```javascript
.prop('checked', true).attr('checked', 'checked')
```